config.md

```diff
@@ -0,0 +1,164 @@
+# The js-ipfs config file
+
+The js-ipfs config file is a JSON document located in the root directory of the js-ipfs repository.
+
+## Table of Contents
+
+- [`Addresses`](#addresses)
+- [`Bootstrap`](#bootstrap)
+- [`Datastore`](#datastore)
+- [`Discovery`](#discovery)
+- [`Identity`](#identity)
+- [`Swarm`](#swarm)
```

Comment:
Do you think it's worth listing sub-sections (just one level more) in the ToC as well?